content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -138,6 +162,56 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+### dvc.lock file
+
+The purpose of `dvc.lock` is to lock the exact contents of all files involved in
```

Comment:
Needs a description for what this lock provides for the `dvc.`.

Reply:
Sorry, I don't understand your comment. The "dvc." ?

Reply:
@skshetry could you summarize it? I mean the value for DVC

Reply:
Something like this, but maybe a bit redundant:

> Using `dvc.lock`, `dvc` can provide support for checkouts for the outputs of the experiments and even allows importing that particular version of the outputs to a different repo via `get`/`import` ...

Reply:
Gotcha. I feel like those benefits can be split between dvc.yaml and dvc.lock really. Also, this part has already changed considerably and we have 3 paragraphs and a bullet list already to explain dvc.lock, so we may have to pick between adding this info or keeping it relatively brief. Up to you @shcheklein please lmk. I'm down to add this info.

Reply:
Ok so per https://github.com/iterative/dvc.org/pull/1448#discussion_r443167630 I went ahead and incorporated this info in d7500f4a. PTAL guys. Resolving here.